Dockerfile

```diff
@@ -41,5 +41,5 @@ RUN git add .
 RUN git diff-index --cached --exit-code HEAD
 
 
-RUN go test -v -timeout 11s ./...
+RUN go test -race -v -timeout 60s ./...
```

Comment:
Timeouts are separate from data races - we had to bump the timeout from 11s because the race detector causes a performance hit (and I pushed it all the way to 60s because I didn't feel like binary-searching my way to the right number for Travis's architecture :) )

I think 60s is a decent timeout, because if any of our tests don't complete in 60s, we want to know about it. Ideally we could keep it down to something even smaller, like 20s, but the performance on the free tier of Travis isn't as good as our local laptops (where the slowest test currently takes <15s for me).